README.md

```diff
@@ -265,9 +265,9 @@ obj3.d === obj.d
 obj3.e === obj.e
 // true
 
-// ... unknown paths create intermediate keys:
-setIn({ a: 3 }, ['unknown', 'path'], 4)
-// { a: 3, unknown: { path: 4 } }
+// ... unknown paths create intermediate keys. Numeric segments are treated as array indices:
```

Comment:
The README is generated automatically based on comments with `// --` in `timm.js`. Please update those as well